apps/app_audiosocket.c

```diff
@@ -180,7 +180,7 @@ static int audiosocket_run(struct ast_channel *chan, const char *id, int svc)
 	chanName = ast_channel_name(chan);
 
 	while (1) {
-
+		ms = 10;
```

Comment:
Can this not just be -1? Then the function will block until there is activity on either the channel or passed in file descriptor.

Reply:
> Can this not just be -1? Then the function will block until there is activity on either the channel or passed in file descriptor.

You're right, -1 is a better choice.